src/Charts.tsx

```diff
@@ -382,19 +382,15 @@ export const Charts: FunctionComponent<ChartsProps> = (props) => {
                     bottom: "35%",
                     zIndex: 4,
                 }}>
-                        <Text size="xl" color={"darkgrey"}>Purchases per Country
-                            <Tooltip label="Purchases per country is tallied starting on page load">
-                                <IconInfoCircle/>
-                            </Tooltip>
```

Comment:
If you're removing this, you can remove the imports for `Tooltip` and `IconInfoCircle` at the top as well